### PR TITLE
[Publish on 15 March 2024] Publish Compliance changes to public-facing Docs

### PR DESCRIPTION
**Pull request by jxainguyen**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by jxainguyen** [comment]
Hey Docs team! 

As usual, please hold off on merging (if everything passes), I'd like to pass this around for QA/sign-off from stakeholders before we merge the PR.

Thx!

**Comment by bradleycamacho** [comment]
Of course @jxainguyen! Just shoot us a message in #help-documentation once ready for review :)

**Comment by tabathadelane** [comment]
Netlify build failed due to a caching issue. Did a cache clear 🔍 [here](https://app.netlify.com/sites/docs-website-netlify/deploys/65f211909af9162cd3cd10b3).

😎 Deploy preview here: https://deploy-preview-16465--docs-website-netlify.netlify.app/

**Comment by jxainguyen** [comment]
Requested another cache update in #help-documentation.

Once the commits clear, just need one quick validation that the nav-bar on the left works when clicking on "ISO-27001". Once that clears, we're good to merge! Thanks team :)

**Comment by rhetoric101** [comment]
@jxainguyen here's the new preview link from the Netlify cloud: https://app.netlify.com/sites/docs-website-netlify/deploys/65f3590349a024202d398c18

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx:151]: @jxainguyen should this "N/A" in the second column be this: AWS, Azure, and GCP?

If so, we'd want to remove the parenthetical (AWS, Azure, and GCP) from the third column, right?

File: src/content/docs/security/security-privacy/compliance/hitrust.mdx
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/compliance/hitrust.mdx:149]: @jxainguyen similar to my question in the other doc, does it make sense to replace the N/A in the second column with this: AWS, Azure, and GCP?

**Review by rhetoric101 — COMMENTED** [review]
@jxainguyen thanks for making these changes! I just had two related questions below. Let me know what you think.

**Comment by rhetoric101** [comment]
@jxainguyen here's the new preview link in the cloud: https://deploy-preview-16465--docs-website-netlify.netlify.app/

**Review by rhetoric101 — APPROVED** [review]
@jxainguyen I went through the docs again and they look good. Thanks for updating these!

**Comment by rhetoric101** [comment]
I just updated this branch from the develop branch to see if that would generate a successful preview check.

**Comment by jxainguyen** [comment]
Done! Thanks a bunch Rob & team for the help :)

**Pull request merged**
